examples/lighting-app/silabs/src/AppTask.cpp

```diff
@@ -175,6 +177,60 @@ void AppTask::LightActionEventHandler(AppEvent * aEvent)
     }
 }
 
+#ifdef SL_MATTER_RGB_LED_ENABLED
+void AppTask::LightControlEventHandler(AppEvent * aEvent)
+{
+    uint8_t light_action = aEvent->LightControlEvent.Action;
+    uint32_t value       = *reinterpret_cast<uint32_t *>(aEvent->LightControlEvent.Value);
+    uint8_t brightness   = 1;
```

Comment:
You can use currentlevel directly. No need for the extra value. If it is Null we should probably not do anything.

Reply:
Modified the code to use the currentLevel and added check for Null or not.